packages/ui/src/scss/normalized.scss

```diff
@@ -2,6 +2,7 @@ body {
   margin: 0;
   padding: 0;
   font-family: sans-serif;
+  word-break: break-word;
```

Comment:
i think it's more like a per-app config? since we are going to import normalized scss from `toolkit`

Reply:
sure, move to a UI specific components